test/jest/config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  collectCoverageFrom: [
+    'ui_framework/components/**/*.js',
+    // Seems to be a bug with jest or micromatch, in which the above glob
+    // doesn't match subsequent levels of directories, making this glob necessary.
+    'ui_framework/components/**/**/*.js',
+    '!ui_framework/components/index.js',
+    '!ui_framework/components/**/*/index.js',
+  ],
+  coverageDirectory: '<rootDir>/target/test_coverage',
```

Comment:
I'll keep this in `target` for now. Looks like the current coverage setup writes to `./coverage`, not something inside the `test` folder. I think both of these should write to `target` over time (well, we'll probably just get rid of the old one when we migrate to Jest).